### PR TITLE
Add desktop wallet articles

### PR DESCRIPTION
**Pull request by Sakrecoer**
Adding a section about Desktop Wallet to the wiki

### CODE REVIEW

File: src/Desktop-Wallet-Guide.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Desktop Wallet Guide
+
+This is the most user-friendly option available for Wallet management. However, some could consider it slightly less privacy-preserving and secure than the node wallet as it requires you to trust the application’s security and is by default connected to a public node. However, you have the possibility to build the wallet directly from our Github repository and to use a local node or even create your wallet completely offline to mitigate some of these risks.
```
Comment (nop33) [inline src/Desktop-Wallet-Guide.md]: What would you say about moving all these privacy and security concerns in their own section somewhere at the end of the article? I personally don't like that the one and only article about the desktop wallet in our wiki starts with concerns...

How about:

> # Desktop Wallet Guide
> 
> The desktop wallet is the most user-friendly Alephium wallet available today. You can download its latest version from our [GitHub](https://github.com/alephium/alephium-wallet/releases/latest).
> 
> ## Step-by-step instructions to install the desktop wallet
> ...
> 
> ## Creating a mining wallet with 4 addresses
> ...
>
> ## UTXO consolidation
> ...
>
> ## Privacy and security concerns
> Some could consider the desktop wallet slightly less privacy-preserving and secure than the node wallet as it requires you to trust the application’s security and is by default connected to a public node. However, you have the possibility to build the wallet directly from our GitHub repository and to use a local node or even create your wallet completely offline to mitigate some of these risks.

File: src/Desktop-Wallet-Guide.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Step-by-step intructions to install the Desktop Wallet
+
+- Go to the [Release Page on GitHub](https://github.com/alephium/alephium-wallet/releases)
```
Comment (nop33) [inline src/Desktop-Wallet-Guide.md]: If you replace this link with https://github.com/alephium/alephium-wallet/releases/latest you always end up to the latest release.

File: src/Desktop-Wallet-Guide.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Download the executable file in line with your set-up (MAC, WINDOWS, Linux). Alternatively, if you wish and are comfortable with coding, you can build the wallet directly from our [Github repository](https://github.com/alephium/alephium-wallet). If you're struggling with this, don't hesitate to contact us on [Discord.](https://discord.gg/JErgRBfRSB)
```
Comment (nop33) [inline src/Desktop-Wallet-Guide.md]: The dot shouldn't be in the link :)

File: src/Desktop-Wallet-Guide.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Desktop Wallet Guide
+
+This is the most user-friendly option available for Wallet management. However, some could consider it slightly less privacy-preserving and secure than the node wallet as it requires you to trust the application’s security and is by default connected to a public node. However, you have the possibility to build the wallet directly from our Github repository and to use a local node or even create your wallet completely offline to mitigate some of these risks.
+
+## Step-by-step intructions to install the Desktop Wallet
+
+- Go to the [Release Page on GitHub](https://github.com/alephium/alephium-wallet/releases)
+- Download the executable file in line with your set-up (MAC, WINDOWS, Linux). Alternatively, if you wish and are comfortable with coding, you can build the wallet directly from our [Github repository](https://github.com/alephium/alephium-wallet). If you're struggling with this, don't hesitate to contact us on [Discord.](https://discord.gg/JErgRBfRSB) 
+- Double click on the file you've downloaded to install it.
+- On Linux and Windows, double click on the application to launch the Desktop wallet.
```
Comment (nop33) [inline src/Desktop-Wallet-Guide.md]: I wouldn't capitalize it: desktop wallet.

File: src/Desktop-Wallet-Guide.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- Click Continue.
+- You now see your new wallet's information 🎉 
+- Your address is a long string of characters. It is what identifies your wallet on the alephium network.
+- In the highlighted box, you will see 24 words. These are your wallet's  24 secret words. This is your most precious piece of information, and you must store it wisely, safely, and carefully.
+- Copy and store your 24 secret words safely on several offline mediums such as paper and metal and store them in different places. 
+- You will now be asked to verify that you got the 24 Secret words right by listing the words in the correct order. Click Ready! And reenter the words with the selection box.
+- When all the words are in the correct order, you will be prompted visually to continue.
+- Everything is ready! Your wallet is now created (and you've made sure to save your 24 words). 
+- To see your public address again, click on “Show address,” copy the string of characters and send it to us by email (Please make sure to use the same email you've listed in your contract).
```
Comment (nop33) [inline src/Desktop-Wallet-Guide.md]: What is this step for? Is it a copy-paste from somewhere? :)

**Review by nop33 — CHANGES_REQUESTED** [review]
Good job @Sakrecoer, thank you for taking care of this!

I've left some comments with some feedback. 

Moreover, I noticed a few typos which would have very easily been caught by a tool like Grammarly which I highly recommend. There are desktop versions and browser extensions as well. I personally use both.

File: src/Desktop-Wallet-Guide.md (hunk 1 above)
Comment (Sakrecoer) [reply src/Desktop-Wallet-Guide.md]: That's brilliant! Thanks! :)

File: src/Desktop-Wallet-Guide.md (hunk 2 above)
Comment (Sakrecoer) [reply src/Desktop-Wallet-Guide.md]: Ufff!! this is  something i've been meaning to find how to achieve! Thanks! :pray:

File: src/Desktop-Wallet-Guide.md (hunk 5 above)
Comment (Sakrecoer) [reply src/Desktop-Wallet-Guide.md]: It is indeed copy-pasted from the first  desktop wallet manual that we ever wrote. I thought i mentioned somewhere this part would probably be inaccurate today, but it looks like i didn't.

**Comment by nop33** [comment]
Hey @Sakrecoer, I've applied the changes I've requested, no need to worry about this :) I'll take care of merging this just before releasing the new version of the wallet.

**Review by nop33 — APPROVED** [review]

**Comment by Sakrecoer** [comment]
Thank you @nop33 !

**Comment by nop33** [comment]
Closing, as content was migrated to https://github.com/alephium/wiki

**Pull request closed without merging**